### PR TITLE
Redirects for home page and unverified users

### LINKED ISSUE #260: Redirect users from root page
The purpose of this page is to give a good first impression to users, but contains no functionality. It is inconvenient to always click one extra, so let's just ease this pain and redirect the user if they are logged in.

### PR DESCRIPTION
**Pull request by luksan47**
I would limit the amount of pages unverified users can access, so with this commit, unverified users won't be able to edit their personal data. This way there is a smaller chance of success for attackers.

Fixes #260 and #225.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -34,17 +38,14 @@
...
 });
 
-Route::middleware(['auth', 'verified'])->group(function () {
-    Route::post('userdata/update_password', 'UserController@updatePassword')->name('userdata.update_password');
-});
-
 Route::middleware(['auth', 'log', 'verified'])->group(function () {
+    Route::get('/home', 'HomeController@index')->name('home');
     Route::get('userdata', 'UserController@showData')->name('userdata');
+    Route::get('/user', 'UserController@index')->name('user');
```
Comment (kdmnk) [inline routes/web.php:47]: Moving user page under verified middleware will bring us back to the issue that unverified users can't log out. An easy solution would be to place a logout button on `auth.verification` blade.

File: routes/web.php [hunk 2]
```diff
@@ -34,17 +38,14 @@
...
+    Route::post('userdata/update_password', 'UserController@updatePassword')->name('userdata.update_password');
```
Comment (kdmnk) [inline routes/web.php:48]: @luksan47 You said updating the password should not be logged.

Comment (luksan47) [reply routes/web.php]: Right, that was a mistake, corrected with the merge commit.

File: routes/web.php (hunk 1 above)
Comment (luksan47) [reply routes/web.php:51]: Right, I thought that's not that important. Added it now, but decided it's good there for unverified users as well, easier to access.

File: resources/views/layouts/navbar.blade.php [hunk 3]
```diff
@@ -56,4 +56,14 @@ function dismiss() {M.Toast.dismissAll();};
     </script>
     <li><a href="mailto:root@eotvos.elte.hu?Subject=[urán%20bug]" onclick="M.toast({html: myHTML, displayLength: 10000})">
             <i class="material-icons left">sentiment_dissatisfied</i>@lang('general.report_bug')</a></li>
+    @if(Auth::user())
+    <li>
+        <a href="{{ route('logout') }}"  onclick="event.preventDefault(); document.getElementById('logout-form').submit();">
+            <i class="material-icons left">login</i>@lang('general.logout')
```
Comment (kdmnk) [inline resources/views/layouts/navbar.blade.php:62]: You could use exit_to_app icon for this (just a little better - it's nonsense that google doesn't have a logout icon :) )

**Pull request merged**